projects/1.20/assets/architects-palette/architects_palette/lang/zh_cn.json

```
@@ -0,0 +1,643 @@
+{
+  "__comment__": "Translators; 'Warps' in this context might be tricky to translate. Even in english its exact meaning is somewhere between teleporting and transforming.",

```

```suggestion
  "__comment__": "Translators; 'Warps' in this context might be tricky to translate. Even in english its exact meaning is somewhere between teleporting and transforming.",
```
根据原作者注释提出建议：warp不应翻译为跃迁，建议改为“移形”，可以兼顾“传送”（取移动意）和“变形”（取改变意）

---

从1.16继承来的翻译之一，已经修改